backend/src/migrations/20240830202033_seed_styles_data.ts

```diff
@@ -0,0 +1,37 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'styles';
+
+async function up(knex: Knex): Promise<void> {
+    await knex(TABLE_NAME).insert([
+        { id: 'b58f9707-e4a4-49fc-8270-5984e70deb70', style: 'business' },
```

Comment:
your seed can fail if you hard code uuid

Reply:
Sorry if I didn't understand, but should I create a JSON file to store the avatar data and manage everything from there? This way, we can keep the configuration static and avoid needing database changes if Azure updates anything.

```
── /avatars
│   ├── /config
│   │   └── avatars.json
│   │
│   ├── /types
│   │
│   ├── /enums
│   │
│   ├── avatar.service.ts
│   │
│   ├── avatar.controller.ts 
│   │
│   └── avatars.ts
```

Reply:
I made the changes